index.js

```diff
@@ -50,6 +55,9 @@ var gulpSass = function gulpSass(options, sync) {
       if (typeof opts.includePaths === 'string') {
         opts.includePaths = [opts.includePaths];
       }
+      else {
+        throw new Error('options.includePaths expected to be a string, ' + typeof opts.includePaths + ' given.');
```

Comment:
I believe the existing behavior is correct. `opts.includePaths` can be either a string or an array of strings.
The type of check coerces a string into an array of one string.
This else branch is the default case. This error will break gulp-sass for most users.